custom_components/meross_cloud/sensor.py

```diff
@@ -287,6 +327,9 @@ def entity_adder_callback():
                     CurrentSensorWrapper(device=d, device_list_coordinator=coordinator, channel=channel_index))
                 new_entities.append(
                     VoltageSensorWrapper(device=d, device_list_coordinator=coordinator, channel=channel_index))
+                if isinstance(d, ConsumptionXMixin):
```

Comment:
Why do you check for the mixin in here (inside the for loop)? I think it would be better to check for the ConsumptionXMixin before the loop and then add the relative sensor wrapper  (as done for humidity/temperature and mts100). In this way, you'll also support devices that only expose the ConsumptionXMixin. If you keep it this way, devices that do not expose the ElectricityXMixin class won't be added.